### PR TITLE
feat(console): active tab by route

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- make console default route to the very first tab
- activate tab by current route
- navigate to route by click

## Linear Issue Reference

LOG-1650

## Testing

image

@logto-io/eng

### CODE REVIEW

File: packages/console/src/App.tsx [hunk 1]
```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { BrowserRouter, useLocation, useNavigate } from 'react-router-dom';
 
 import './scss/normalized.scss';
 import * as styles from './App.module.scss';
 import Content from './components/Content';
 import Sidebar from './components/Sidebar';
+import { sections } from './components/Sidebar/consts';
+import { getPath } from './components/Sidebar/utils';
```
Comment (simeng-li) [inline packages/console/src/App.tsx:9]: nit: would it be handier to make them all exportable through Sidebar Components
import. Sidebar, { sections, getPath } from './components/Sidebar`?

**Review by simeng-li — APPROVED** [review]
overall LGTM

File: packages/console/src/components/Sidebar/components/Item/index.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import classNames from 'classnames';
 import React, { ReactChild } from 'react';
 
+import { getPath } from '../../utils';
```
Comment (demonzoo) [inline packages/console/src/components/Sidebar/components/Item/index.tsx:4]: Use `@` alias?

**Review by demonzoo — APPROVED** [review]

File: packages/console/src/App.tsx (hunk 1 above)
Comment (gao-sun) [reply packages/console/src/App.tsx:9]: cool

**Pull request merged**